examples/cookbook/recipe_move_between_numa.cpp

```diff
@@ -26,7 +27,7 @@ int main(int, char**)
   auto host_nodes = umpire::numa::get_host_nodes();
 
   if (host_nodes.size() < 1) {
-    UMPIRE_ERROR("No NUMA nodes detected");
+    UMPIRE_ERROR(umpire::runtime_error, umpire::fmt::format("No NUMA nodes detected: {}", host_nodes.size()));
```

Comment:
```suggestion
    UMPIRE_ERROR(umpire::runtime_error, "No NUMA nodes detected");
```

Reply:
Can I redeem myself by saying I was multitasking when doing this? Lol your suggestion seems much more reasonable here than what I was about to do!